package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@infernored/electionguard-experimental",
+    "version": "0.0.5",
+    "description": "Library for encrypting ballots for the ElectionGuard voting system",
+    "author": "Microsoft <electionguard@microsoft.com>",
+    "license": "MIT",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/microsoft/electionguard-core2.git"
+    },
+    "bugs": {
+        "url": "https://github.com/microsoft/electionguard-core2/issues"
+    },
+    "main": "dist/index.js",
```

Comment:
i thought about wether or not to put this npm stuff in the root or to put it in the `bindings/typescript` folder. i didnt come up with a good justification either way. i could be convinced to move it but it would be done in another PR